src/quoridor/game/game.go

```diff
@@ -27,6 +45,7 @@ func (g Game) AddFence(fence Fence) (Game, error) {
 	if err != nil {
 		return Game{}, err
 	}
+	g.PlayerTurn = g.whoIsNext()
```

Comment:
"whoIsNext" is not very relevant, can you rename it with a more meaning name like getNextPlayerTurn, etc...